patches/@polkadot+apps-config+0.102.1.patch

```diff
@@ -1,22 +0,0 @@
-diff --git a/node_modules/@polkadot/apps-config/endpoints/index.js b/node_modules/@polkadot/apps-config/endpoints/index.js
-index 91e3d2e..871b8a4 100644
---- a/node_modules/@polkadot/apps-config/endpoints/index.js
-+++ b/node_modules/@polkadot/apps-config/endpoints/index.js
-@@ -1,6 +1,6 @@
- // Copyright 2017-2022 @polkadot/apps-config authors & contributors
- // SPDX-License-Identifier: Apache-2.0
--import { createCustom, createDev, createOwn } from "./development.js";
-+import { createCustom, createDev } from "./development.js";
- import { createProduction } from "./production.js";
- import { createKusamaRelay, createPolkadotRelay } from "./productionRelays.js";
- import { createTesting } from "./testing.js";
-@@ -68,7 +68,7 @@ export function createWsEndpoints(t, firstOnly = false, withSort = true) {
-     }),
-     textBy: '',
-     value: ''
--  }, ...createDev(t), ...createOwn(t)].filter(({
-+  }, ...createDev(t)].filter(({
```

Comment:
Is this not needed anymore?

Reply:
The patch is not needed anymore. They included checks.